apps/dashboard/app/(app)/semantic-cache/new/page.tsx

```diff
@@ -23,11 +24,7 @@ export default async function NewSemanticCachePage() {
     return redirect(`/semantic-cache/${workspace.llmGateways[0].id}/logs`);
   }
 
-  const defaultName = `${faker.hacker.adjective()}-${faker.hacker.adjective()}-${
-    faker.science.chemicalElement().name
-  }-${faker.number.int({ min: 1000, max: 9999 })}`
-    .replaceAll(/\s+/g, "-")
-    .toLowerCase();
+  const defaultName = generateSemanticCacheDefaultName();
```

Comment:
I also think this should be similar to what happens now, when you are already signed in / up. 

The user should be able to choose the LLM gateway subdomain but have it automatically filed with a generated one versus getting dumped into one. This is personal preference but feels like if you wanted to onboard because you have an LLM you'd choose something like "my-app-name". 

Which means because you can't edit the LLM gateway you'd have to delete it and restart.

Similar to this `apps/dashboard/app/(app)/gateways/new/form.tsx` which shows as this https://app.unkey.com/semantic-cache/new

Reply:
I do agree with this, will work in some second screen/step just before deploying a new gateway